src/GuideAtom.stories.tsx

```diff
@@ -16,6 +16,82 @@ export const DefaultStory = (): JSX.Element => {
+            credit=""
+            likeHandler={() => {
+                return null;
+            }}
+            dislikeHandler={() => {
+                return null;
+            }}
+            expandHandler={() => {
+                return null;
+            }}
+        />
+    );
+};
+
+export const OrderedListStory = (): JSX.Element => {
+    return (
+        <GuideAtom
+            id="b4c77875-8ba3-40d0-926b-0cd7956eed8a"
+            label="Quick Guide"
+            title="The locations in England with the highest annual average levels of NO2"
+            html="<ol><li>Chideock Hill, West Dorset 97.7</li><li>Station Taxi Rank, Sheffield 91.7</li><li>North Street Clock Tower, Brighton 90.8</li><li>Neville Street Tunnel, Leeds 88</li><li>Strand, City of Westminster 88</li><li>Walbrook Wharf, City of London 87</li><li>Hickleton opp Fir Tree Close, Doncaster 86</li><li>Marylebone Road, City of Westminster 85</li><li>Euston Road, London Borough of Camden 82.3</li><li>Hickleton, John O’Gaunts, Doncaster 82</li></ol><p>&nbsp;Latest data is from 2018.&nbsp; The Annual Air Quality Objective is set at 40ug/m3. Source: Friends of the Earth&nbsp;<br></p>"
+            credit=""
+            likeHandler={() => {
+                return null;
+            }}
+            dislikeHandler={() => {
+                return null;
+            }}
+            expandHandler={() => {
+                return null;
+            }}
+        />
+    );
+};
+
+export const ImageStory = (): JSX.Element => {
```

Comment:
These seem to be the main styles used inside of guide atoms, so I thought it would be useful to have an example for each. Mainly to check that list styles or ul and ol don't interfere with each other. It's also useful to check one with an image to make sure the logic works correctly in terms of spacing there.